packages/framework/ini-files/config-specs.ini

```diff
@@ -950,6 +950,184 @@ opt-set-cmake-var Trilinos_PARALLEL_LINK_JOBS_LIMIT STRING FORCE : 16
+opt-set-cmake-var ROL_example_PDE-OPT_0ld_poisson_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_0ld_stefan-boltzmann_example_03_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_navier-stokes_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_navier-stokes_example_02_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_nonlinear-elliptic_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_nonlinear-elliptic_example_02_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_obstacle_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_poisson-boltzmann_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_example_PDE-OPT_topo-opt_poisson_example_01_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var ROL_test_algorithm_TypeB_TrustRegionSPG_MPI_1_DISABLE BOOL : ON
+opt-set-cmake-var ROL_test_elementwise_TpetraMultiVector_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_aprepro_array_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_aprepro_command_line_include_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_aprepro_command_line_vars_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_aprepro_test_dump_reread_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_aprepro_unit_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_lib_aprepro_lib_array_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASAprepro_lib_aprepro_lib_unit_test_DISABLE BOOL : ON
+opt-set-cmake-var SEACASIoss_create_path_fpp_DISABLE BOOL : ON
+opt-set-cmake-var STKUnit_tests_stk_ngp_test_utest_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var STKUnit_tests_stk_mesh_unit_tests_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var Stokhos_TpetraCrsMatrixMPVectorUnitTest_Cuda_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var Teko_ModALPreconditioner_MPI_1_DISABLE BOOL : ON
+opt-set-cmake-var TrilinosCouplings_Example_Maxwell_MueLu_MPI_1_DISABLE BOOL : ON
+opt-set-cmake-var TrilinosCouplings_Example_Maxwell_MueLu_MPI_4_DISABLE BOOL : ON
+opt-set-cmake-var Zoltan_ch_simple_zoltan_parallel_DISABLE BOOL : ON
+opt-set-cmake-var Zoltan_hg_simple_zoltan_parallel_DISABLE BOOL : ON
+opt-set-cmake-var Zoltan_mpiMinLoc_MPI_4_DISABLE BOOL : ON
+# These disables are due to the problem described in: https://github.com/trilinos/Trilinos/pull/10253#issuecomment-1088066182
+opt-set-cmake-var Compadre_GMLS_Staggered_Dim2_QR_MPI_1_DISABLE BOOL : ON
+opt-set-cmake-var Compadre_GMLS_Staggered_Dim3_QR_MPI_1_DISABLE BOOL : ON
+opt-set-cmake-var Compadre_GMLS_Staggered_Manifold_Refinement_Study_DISABLE BOOL : ON
+
+
+[ats2_cuda-11.2.152-gnu-8.3.1-spmpi-rolling_release_static_Volta70_Power9_no-asan_no-complex_no-fpic_mpi_pt_no-rdc_no-uvm_deprecated-on_all]
+use ATS2
+use ATS2-COMMON-OVERRIDES
+
+use NODE-TYPE|CUDA
+use BUILD-TYPE|RELEASE
+# Override settings from [BUILD-TYPE|RELEASE]
+opt-set-cmake-var Kokkos_ENABLE_DEBUG_BOUNDS_CHECK BOOL FORCE : ON
```

Comment:
I assume this was restored from the previous build config, but do we really want Kokkos debugging enabled?  If it's just restored from old build, then we can leave it.